packages/salesforcedx-utils-vscode/src/settings/advancedSettings.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export enum AdvancedSettings {
+  ENABLE_LOCAL_TELEMETRY_LOGGING = 'enableLocalTelemetryLogging'
```

Comment:
if only developers are setting this, we could shorten the name. Setting to `true` could help remove ambiguity in the name too, e.g. `localTelemetryLogging:true`=> implies it's enabled